packages/core/src/shaderlib/pbr/pbr_helper.glsl

```diff
@@ -1,44 +1,75 @@
 #include <normal_get>
 
 
-float pow2(float x ) {
-    return x * x;
+float computeSpecularOcclusion(float ambientOcclusion, float roughness, float dotNV ) {
+    return saturate( pow( dotNV + ambientOcclusion, exp2( - 16.0 * roughness - 1.0 ) ) - 1.0 + ambientOcclusion );
 }
 
-vec3 BRDF_Diffuse_Lambert(vec3 diffuseColor ) {
-	return RECIPROCAL_PI * diffuseColor;
+float getAARoughnessFactor(vec3 normal) {
+    #ifdef HAS_DERIVATIVES
+        vec3 dxy = max( abs(dFdx(normal)), abs(dFdy(normal)) );
+        return max( max(dxy.x, dxy.y), dxy.z );
+    #else
+        return 0.0;
+    #endif
 }
```

Comment:
If don't have `HAS_DERIVATIVES` ,not use `0.04` to project?

Reply:
done